setup/inject_SELinux_AVC_check/main.fmf

```diff
@@ -0,0 +1,15 @@
+summary: Inject AVC SELinux checking into beakerlib phases
+description: |
+        Inject into beakerlib functions rlPhaseStartTest, rlPhaseStartSetup setting timestamp, 
+        and to rlPhaseEnd checking AVC based on timestamp set before. Unset timestamp variable 
+        in rlPhaseStartCleanup due to unecessary checking of AVC's in clean up phase.
+contact: Patrik Koncity <pkoncity@redhat.com>
+component:
+- keylime
+test: ./test.sh
+tag:
+ - setup
+framework: beakerlib
```

Comment:
We wanted to add requirements on `audit`. I believe it makes sense to be explicit, even though we may expect it to be installed by default.